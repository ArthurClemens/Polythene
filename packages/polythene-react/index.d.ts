export { Button } from "polythene-react-button";
export { ButtonGroup } from "polythene-react-button-group";
export { Card } from "polythene-react-card";
export { Checkbox } from "polythene-react-checkbox";
export { Dialog } from "polythene-react-dialog";
export { DialogPane } from "polythene-react-dialog-pane";
export { Drawer } from "polythene-react-drawer";
export { FAB } from "polythene-react-fab";
export { Icon } from "polythene-react-icon";
export { IconButton } from "polythene-react-icon-button";
export { IOSSpinner } from "polythene-react-ios-spinner";
export { List } from "polythene-react-list";
export { ListTile } from "polythene-react-list-tile";
export { MaterialDesignProgressSpinner } from "polythene-react-material-design-progress-spinner";
export { MaterialDesignSpinner } from "polythene-react-material-design-spinner";
export { Menu } from "polythene-react-menu";
export { Notification } from "polythene-react-notification";
export { RadioButton } from "polythene-react-radio-button";
export { RadioGroup } from "polythene-react-radio-group";
export { Ripple } from "polythene-react-ripple";
export { Search } from "polythene-react-search";
export { Shadow } from "polythene-react-shadow";
export { Slider } from "polythene-react-slider";
export { Snackbar } from "polythene-react-snackbar";
export { SVG } from "polythene-react-svg";
export { Switch } from "polythene-react-switch";
export { Tabs } from "polythene-react-tabs";
export { TextField, onChangeTextFieldState } from "polythene-react-textfield";
export { Toolbar, ToolbarTitle } from "polythene-react-toolbar";
