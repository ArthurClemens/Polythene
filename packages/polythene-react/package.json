{
  "name": "polythene-react",
  "version": "1.3.3",
  "description": "",
  "main": "dist/polythene-react",
  "module": "dist/polythene-react.mjs",
  "scripts": {
    "clean": "rm -rf dist/*",
    "rollup": "../../node_modules/rollup/bin/rollup -c ../../scripts/bundling/rollup.umd.js && ../../node_modules/rollup/bin/rollup -c ../../scripts/bundling/rollup.es.js",
    "build-standalone": "ENTRY=scripts/standalone.js DEST=./dist/polythene-react-standalone ../../node_modules/rollup/bin/rollup -c ../../scripts/bundling/rollup.browser.js",
    "build": "npm run clean && npm run rollup && npm run build-standalone"
  },
  "files": [
    "dist",
    "README.md"
  ],
  "dependencies": {
    "polythene-core-css": "^1.3.3",
    "polythene-core": "^1.3.3",
    "polythene-css": "^1.3.3",
    "polythene-react-base": "^1.3.3",
    "polythene-react-button-group": "^1.3.3",
    "polythene-react-button": "^1.3.3",
    "polythene-react-card": "^1.3.3",
    "polythene-react-checkbox": "^1.3.3",
    "polythene-react-dialog-pane": "^1.3.3",
    "polythene-react-dialog": "^1.3.3",
    "polythene-react-drawer": "^1.3.3",
    "polythene-react-fab": "^1.3.3",
    "polythene-react-icon-button": "^1.3.3",
    "polythene-react-icon": "^1.3.3",
    "polythene-react-ios-spinner": "^1.3.3",
    "polythene-react-list-tile": "^1.3.3",
    "polythene-react-list": "^1.3.3",
    "polythene-react-material-design-progress-spinner": "^1.3.3",
    "polythene-react-material-design-spinner": "^1.3.3",
    "polythene-react-menu": "^1.3.3",
    "polythene-react-notification": "^1.3.3",
    "polythene-react-radio-button": "^1.3.3",
    "polythene-react-radio-group": "^1.3.3",
    "polythene-react-raised-button": "^1.3.3",
    "polythene-react-ripple": "^1.3.3",
    "polythene-react-search": "^1.3.3",
    "polythene-react-shadow": "^1.3.3",
    "polythene-react-slider": "^1.3.3",
    "polythene-react-snackbar": "^1.3.3",
    "polythene-react-svg": "^1.3.3",
    "polythene-react-switch": "^1.3.3",
    "polythene-react-tabs": "^1.3.3",
    "polythene-react-textfield": "^1.3.3",
    "polythene-react-toolbar": "^1.3.3"
  },
  "peerDependencies": {
    "react": "^16.2.0",
    "react-hyperscript": "^3.0.0"
  },
  "author": "Arthur Clemens <arthurclemens@gmail.com> (http://arthurclemens.com)",
  "homepage": "https://github.com/ArthurClemens/polythene",
  "license": "MIT"
}
