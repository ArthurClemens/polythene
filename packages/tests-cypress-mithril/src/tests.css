.page__test__component .align-items-center {
  display: flex;
  align-items: center;
}