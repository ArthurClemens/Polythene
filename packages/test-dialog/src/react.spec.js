import { runSnapshots } from "../../../scripts/tests/react-snapshots";
import reactTests from "./tests-react.js";
import { renderer, keys, Button, Dialog, DialogInstance } from "polythene-react";
import specTests from "./spec-tests.js";

runSnapshots({
  tests: specTests({ renderer, keys, Button, Dialog, DialogInstance }).concat(reactTests),
  renderer
});
