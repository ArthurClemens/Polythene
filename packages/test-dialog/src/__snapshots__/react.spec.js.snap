// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No options - spec 1`] = `
<div
  className="pe-dialog    "
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={3}
    />
    <DialogPane />
  </div>
</div>
`;

exports[`Option: body - spec 1`] = `
<div
  className="pe-dialog    "
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={3}
    />
    <DialogPane
      body="Hello"
    />
  </div>
</div>
`;

exports[`Option: class - spec 1`] = `
<div
  className="pe-dialog    class-x"
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={3}
    />
    <DialogPane
      className="class-x"
    />
  </div>
</div>
`;

exports[`Option: className - spec 1`] = `
<div
  className="pe-dialog    className-x"
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={3}
    />
    <DialogPane
      className="className-x"
    />
  </div>
</div>
`;

exports[`Option: element - spec 1`] = `
<a
  className="pe-dialog    "
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={3}
    />
    <DialogPane />
  </div>
</a>
`;

exports[`Option: id - spec 1`] = `
<div
  className="pe-dialog    "
  id="id-x"
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={3}
    />
    <DialogPane />
  </div>
</div>
`;

exports[`Option: modal with backdrop - spec 1`] = `
<div
  className="pe-dialog    "
  onClick={[Function]}
>
  <div
    className="pe-dialog__backdrop"
    key="backdrop"
  />
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={3}
    />
    <DialogPane
      body={
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>",
            }
          }
        />
      }
      footerButtons={
        Array [
          <Button
            events={
              Object {
                "onClick": [Function],
              }
            }
            label="Cancel"
          />,
          <Button
            events={
              Object {
                "onClick": [Function],
              }
            }
            label="Save"
          />,
        ]
      }
      title="Long dialog with a very long title that surely won't fit here"
    />
  </div>
</div>
`;

exports[`Option: title and body (long text) - spec 1`] = `
<div
  className="pe-dialog    "
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={3}
    />
    <DialogPane
      body={
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>",
            }
          }
        />
      }
      title="Long dialog with a very long title that surely won't fit here"
    />
  </div>
</div>
`;

exports[`Option: z (0) - spec 1`] = `
<div
  className="pe-dialog    "
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={0}
    />
    <DialogPane
      body={
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>",
            }
          }
        />
      }
    />
  </div>
</div>
`;

exports[`Option: z (5) - spec 1`] = `
<div
  className="pe-dialog    "
  onClick={[Function]}
>
  <div
    className="pe-dialog__touch"
    key="touch"
  />
  <div
    className="pe-dialog__content "
    key="content"
  >
    <Shadow
      animated={true}
      key="shadow"
      z={6}
    />
    <DialogPane
      body={
        <div
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>",
            }
          }
        />
      }
    />
  </div>
</div>
`;
